scanners/github-octokit-checks/graphql-queries.js

```diff
@@ -0,0 +1,134 @@
+// https://www.npmjs.com/package/@octokit/graphql
```

Comment:
What is the purpose of this module?

If this is for reference/documentation purposes, my suggestion is to create a `.md` file with some embedded code that can be referenced in the future.

Reply:
Oh no - that was a mistake commit - it was my working through moving the query from the explorer over to node, before you mentioned we can use straight octokit.